src/lib/syntax/ast/expr.rs

```diff
@@ -200,6 +198,13 @@ impl Display for ExprDef {
                 }
                 f.write_str("")
             }
+            ExprDef::ConstDeclExpr(ref vars) => {
+                f.write_str("var ")?;
```

Comment:
does this need to be `f.write_str("const ")?;` ?

Reply:
Ups, yes, I copied it from above.